### PR TITLE
feat: sync user wall data

### LINKED ISSUE #228: sync users wall on the homepage
Hi, since the update of the user's wall on the homepage last time, there have more than 2+ users.

[1] https://github.com/apache/apisix/issues/487?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDU4MDI1Mzk3MzoyMTA2OTg3&notifications_query=repo%3Aapache%2Fapisix#issuecomment-773107772

### PR DESCRIPTION
**Pull request by stu01509**
Fixes: #228

Changes:
Add the 腾讯游戏, but I am not sure how to upload the logo into  **apiseven.com**

### CODE REVIEW

**Comment by juzhiyuan** [comment]
@stu01509 Hi, could we keep those images in our repo? There has an issue to track this 😊 #196

**Review by tokers — APPROVED** [review]

**Comment by stu01509** [comment]
Hi @juzhiyuan 

Ok, but I think maybe we can merge this is PR, I will move all static files under this repo in the next PR

**Comment by juzhiyuan** [comment]
image

Hi, this logo's text is the same as background, then users will not know it's brand.

**Comment by juzhiyuan** [comment]
image

You could visit this preview link to see what happened here.

**Comment by stu01509** [comment]
Hi @juzhiyuan 

Thanks for your mention, so what kind of type should I replace it?

**Comment by stu01509** [comment]
Hi @juzhiyuan 

Update the commit, please review it :)

File: website/static/data/showcases.json [hunk 1]
```diff
@@ -1,260 +1,260 @@
 [
   {
     "caption": "360 奇虎",
-    "image": "https://static.apiseven.com/logo/360.png",
+    "image": "./../img/showcase/360.pnh",
```
Comment (juzhiyuan) [inline website/static/data/showcases.json]: Could we use `../img/xxx`?

**Comment by juzhiyuan** [comment]
and I noticed that, there has some broken links:

image

Please take a look here: https://deploy-preview-260--apisix.netlify.app/

**Comment by stu01509** [comment]
> and I noticed that, there has some broken links:
> 
> image
> 
> Please take a look here: https://deploy-preview-260--apisix.netlify.app/

Update the commit :)

**Comment by juzhiyuan** [comment]
Hi, LGTM, but do we really need ./../xxx? How about../xxx

**Comment by stu01509** [comment]
> Hi, LGTM, but do we really need ./../xxx? How about../xxx

OK, update the commit.

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**